tests/sklearn/test_sklearn_model_export.py

```diff
@@ -747,28 +747,13 @@ def test_sklearn_compatible_with_mlflow_2_4_0(sklearn_knn_model, tmp_path):
     ).values.squeeze()
     np.testing.assert_array_almost_equal(scores, model_predict)
 
-    # Raise error if trying to pass params to model logged with mlflow < 2.5.0
-    with pytest.raises(
-        MlflowException,
-        match=r"`params` can only be specified at inference "
-        r"time if the model signature defines a params schema.",
-    ):
+    # Issues a warning if params are specified prior to MLflow support in 2.5.0
+    with mock.patch("mlflow.models.utils._logger.warning") as mock_warning:
         pyfunc_loaded.predict(inference_dataframe, params={"top_k": 2})
-
-    # Raise error if trying to pass params to model logged with mlflow < 2.5.0 for model serving
```

Comment:
Is there a way to warn users about ignoring the params in this case if they use model serving?

Reply:
I don't think out good way, user have to check server log.
if we add warning message into response content, it breaks response data of prediction result